relays/ethereum/src/instances.rs

```diff
@@ -44,9 +44,9 @@ pub trait BridgeInstance: Send + Sync + std::fmt::Debug {
 
 /// Corresponds to the Rialto instance used in the bridge runtime.
 #[derive(Default, Clone, Debug)]
-pub struct Rialto;
+pub struct TestPoA;
```

Comment:
To avoid confusion - there's type with the same `substrate-client`.

Reply:
I'd prefer to change the type in `substrate-client` to `RialtoRuntime` and keep this instance type as `Rialto`

Reply:
I'll change this to RialtoPoA for now - if you'll decide to use something else, it could be changed. So now (in near future) we'll have 3 chains:
- Millau-Sub - chain that has all modules for sub2sub sync with Rialto-Sub;
- Rialto-Sub - chain that has all modules for sub2sub sync with Millau-Sub and all modules to sync with Rialto-PoA;
- Rialto-PoA - Etherum PoA chain.

For simplicity, I suggest to use Millau/Rialto both to identify bridges and their 'main' substrate-based chains. Other chains should be referenced with some suffix - e.g. RialtoPoA. Or we may use some names as you've suggested, but it isn't a part of this PR.